packages/smooth_app/lib/pages/image_crop_page.dart

```diff
@@ -6,137 +6,153 @@ import 'package:image_cropper/image_cropper.dart';
 import 'package:image_picker/image_picker.dart';
 import 'package:smooth_app/generic_lib/dialogs/smooth_alert_dialog.dart';
 
+/// Returns the file path of an image after it's been cropped.
+///
+/// This is the "old" problematic version; to be rapidly changed.
+Future<String?> getCroppedPath(
+  final BuildContext context,
+  final String inputPath,
+) async =>
+    (await ImageCropper().cropImage(
+      sourcePath: inputPath,
+      aspectRatioPresets: <CropAspectRatioPreset>[
+        CropAspectRatioPreset.square,
+        CropAspectRatioPreset.ratio3x2,
+        CropAspectRatioPreset.original,
+        CropAspectRatioPreset.ratio4x3,
+        CropAspectRatioPreset.ratio16x9
+      ],
+      uiSettings: <PlatformUiSettings>[
+        AndroidUiSettings(
+          initAspectRatio: CropAspectRatioPreset.original,
+          lockAspectRatio: false,
+          toolbarTitle: AppLocalizations.of(context).product_edit_photo_title,
+          // They all need to be the same for dark/light mode as we can't change
+          // the background color and the action bar color
+          statusBarColor: Colors.black,
+          toolbarWidgetColor: Colors.black,
+          backgroundColor: Colors.black,
+          activeControlsWidgetColor: const Color(0xFF85746C),
+        ),
+      ],
+    ))
+        ?.path;
+
+/// Crops an image from an existing file.
+Future<File?> startImageCroppingNoPick(
+  final BuildContext context, {
+  required final File existingImage,
+}) async {
+  final NavigatorState navigator = Navigator.of(context);
+  await _showScreenBetween(navigator);
+
+  // ignore: use_build_context_synchronously
+  final String? croppedPath = await getCroppedPath(
+    context,
+    existingImage.path,
+  );
+
+  await _hideScreenBetween(navigator);
+
+  if (croppedPath == null) {
+    return null;
+  }
+
+  return File(croppedPath);
+}
+
+/// Picks an image file from gallery or camera.
+Future<XFile?> pickImageFile(
+  final BuildContext context, {
+  final bool showOptionDialog = false,
+  bool chooseFromGallery = false,
+}) async {
+  if (showOptionDialog) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+    final bool? dialogFromGallery = await showDialog<bool>(
+      context: context,
+      builder: (BuildContext context) => SmoothAlertDialog(
```

Comment:
Could you add the axis to vertical for the buttons?

Reply:
Sure. Fixed in the latest push.